lmdk/include/buffer.h

```diff
@@ -140,7 +140,7 @@ struct comp_buffer {
 	uint32_t pipeline_id;
 	uint32_t caps;
 	uint32_t core;
-//	struct tr_ctx tctx;			/* trace settings */
+	uint32_t padd[2];
```

Comment:
I don't get it, are you trying to have 2 copies of the same structure, with some fields as padds? 
Maintenance of this will be extremely hard

Reply:
ideally both base FW and loadable moule use the same structure definition so that they can decode structure correctly. But Now loadable module remove above tctx for trace. This result to inconsistency of the the structure and decoding can't be correct.  Add padd data to avoid structure size and internal offset change.